crates/chia-consensus/src/spendbundle_conditions.rs

```diff
@@ -0,0 +1,326 @@
+use crate::consensus_constants::ConsensusConstants;
+use crate::gen::conditions::{
+    process_single_spend, validate_conditions, MempoolVisitor, ParseState, SpendBundleConditions,
+};
+use crate::gen::flags::MEMPOOL_MODE;
+use crate::gen::run_block_generator::subtract_cost;
+use crate::gen::validation_error::ValidationErr;
+use crate::spendbundle_validation::get_flags_for_height_and_constants;
+use chia_protocol::SpendBundle;
+use clvm_utils::tree_hash;
+use clvmr::allocator::Allocator;
+use clvmr::chia_dialect::ChiaDialect;
+use clvmr::reduction::Reduction;
+use clvmr::run_program::run_program;
+use clvmr::serde::node_from_bytes;
+
+pub fn get_conditions_from_spendbundle(
+    a: &mut Allocator,
+    spend_bundle: &SpendBundle,
```

Comment:
Can this take `impl IntoIterator<Item = impl Borrow<CoinSpend>>` instead of `&SpendBundle` so that it doesn't require the `aggregated_signature` to be passed in?

Reply:
I suppose it would be more aptly named `get_conditions_from_coinspends` in this case